fluentui_tablayout/src/main/java/com/microsoft/fluentui/tablayout/TabLayout.kt

```diff
@@ -49,28 +48,29 @@ class TabLayout : TemplateView {
     private val tabSelectedTextColor: Int
     private val tabUnselectedTextColor: Int
 
+
     @JvmOverloads
-    constructor(context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0) : super(FluentUIContextThemeWrapper(context), attrs, defStyleAttr) {
+    constructor(context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0) : super(FluentUIContextThemeWrapper(context,R.style.Theme_FluentUI_TabLayout), attrs, defStyleAttr) {
         val styledAttributes = context.obtainStyledAttributes(attrs, R.styleable.TabLayout)
         val tabTypeOrdinal = styledAttributes.getInt(R.styleable.TabLayout_tabType, TabType.STANDARD.ordinal)
         containerBackgroundColor = styledAttributes.getColor(
             R.styleable.TabLayout_containerBackgroundColor,
-            ThemeUtil.getColor(context, R.attr.fluentuiTabLayoutContainerBackgroundColor))
+            ThemeUtil.getColor(this.context, R.attr.fluentuiTabLayoutContainerBackgroundColor))
```

Comment:
add proper comments to explain this.context, will be helpful for other people contributing to this

Reply:
Done. Fixed in next patch